frontend/src/pages/main/ContactUs.js

```diff
@@ -0,0 +1,86 @@
+import "./styles/ContactUs.css";
+import Map from "../../components/Map";
+import { useState } from "react";
+const ContactUs = () => {
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+  const [email, setEmail] = useState('')
+  const [selectedCity, setSelectedCity] = useState('');
+  const [userMessage, setUserMessage] = useState('');
+  const NAME_regEx = '^[A-Za-z]+';
+  const MAIL_regEx = '^[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,}$';
+
+  const cites = [
```

Comment:
correct the typo `cites` -> `cities` here and everywhere it is used in code below.